src/Data/testimonialData.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export const hiitTribe = {
+  message: (
+    <span>
+      My experience working with the GoDragons team was <b>remarkable</b>!
+      Milestones were respected with accurate implementation of specs,
+      improvements were well articulated, responsiveness was prompt, issues were
+      resolved without unnecessary pushbacks, and completion stayed{" "}
+      <b>within budget</b>. I will be working with GoDragons for{" "}
+      <b>many years</b> to come!{" "}
+    </span>
+  ),
+  author: "Alex - CEO at HIIT Tribe",
```

Comment:
shall we split this into `name` and `position`? maybe that is overengineering... but I think it will be the same format each time.